src/service.py

```diff
@@ -1,17 +1,60 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 import os
+import uuid
 from dotenv import load_dotenv
+from typing import Generator, Sequence
+from sqlalchemy.orm import Session
+from sqlalchemy import select
+from src.models import Projects
+from src.schemas import Project
 
 load_dotenv()
 
-POSTGRES_USER = os.environ.get("POSTGRES_USER")
-POSTGRES_PASSWORD = os.environ.get("POSTGRES_PASSWORD")
-POSTGRES_DB = os.environ.get("POSTGRES_DB")
-POSTGRES_PORT = os.environ.get("POSTGRES_PORT")
-POSTGRES_SERVER = os.environ.get("POSTGRES_SERVER")
+POSTGRES_USER = os.environ.get("POSTGRES_USER", "test_user")
+POSTGRES_PASSWORD = os.environ.get("POSTGRES_PASSWORD", "test_password")
+POSTGRES_DB = os.environ.get("POSTGRES_DB", "test_db")
+POSTGRES_PORT = os.environ.get("POSTGRES_PORT", "5432")
+POSTGRES_SERVER = os.environ.get("POSTGRES_SERVER", "localhost")
```

Comment:
for user, password and db name you shouldn't have such default values. either make them empty strings or remove completely